.jenkins/actions/test_radiation.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# This scripts provides a Jenkins action to run and validate the radiation scheme
+# with all relevant GT4Py backends. Several environment variables (see below) are exptected
+# to be set upon execution.
+
+# 2021/09/24 Andrew Pauling, Allen AI, andrewp@allenai.org
+
+# stop on all errors (also within a pipe-redirection)
+set -e -x
+set -o pipefail
+
+# the following environment variables need to be set
+#  scheme  -  the radiation scheme to test (SW or LW)
+#                       (note: parameterization="buildenv" is used to setup environment)
+#  backend           -  the GT4Py backend to run
+if [ -z "${scheme}" ] ; then
+    echo "ERROR: the variable 'scheme' needs to be set to run this Jenkins action"
+    exit 1
+fi
+if [ -z "${backend}" ] ; then
+    echo "ERROR: the variable 'backend' needs to be set to run this Jenkins action"
+    exit 1
+fi
+
+echo "Information:"
+echo "  JOB_NAME=${JOB_NAME}"
+echo "  BUILD_NUMBER=${BUILD_NUMBER}"
+echo "  WORKSPACE=${WORKSPACE}"
+echo "  parameterization=${parameterization}"
+echo "  backend=${backend}"
+echo "  venv=${venv}"
+echo ""
+
+if [ "${backend}" == "numpy" ] && [ "${parameterization}" == "buildenv" ] ; then
```

Comment:
why is this for the numpy backend specifically? because that's what the other physics packages have set up?

Reply:
I just left this in from the test script Elynn had for the physics. The radiation scheme doesn't actually work with the numpy backend yet.